SpatialGDK/Source/SpatialGDK/Private/Tests/SpatialView/EntityComponentTestUtils.h

```diff
@@ -0,0 +1,180 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#pragma once
+
+#include "SpatialView/EntityComponentTypes.h"
+
+namespace SpatialGDK
+{
+
+static const Schema_FieldId kEventId = 1;
+static const Schema_FieldId kEventIntFieldId = 2;
+
+static const Schema_FieldId kTestDoubleFieldId = 1;
+
+inline ComponentData CreateTestComponentData(Worker_ComponentId id, double value)
+{
+	ComponentData componentData{ id };
```

Comment:
Is there some way we can Capitalise the start of "componentData". I understand you are differentiating from the Type name.

Reply:
You're right, it should be capitalised 👍

Reply:
Fixed.